awx_collection/plugins/doc_fragments/auth_plugin.py

```diff
@@ -18,42 +18,42 @@ class ModuleDocFragment(object):
     env:
     - name: CONTROLLER_HOST
     - name: TOWER_HOST
-    deprecated:
-    - removed_from_collection: 'awx.awx.controller'
-    - removed_in: '4.0.0'
-    - why: Collection name change
-    - alternative: 'CONTROLLER_HOST'
+      deprecated:
+        removed_from_collection: 'awx.awx'
```

Comment:
And some more things that ansible-core expects (while antsibull-docs simply accepts them):
```suggestion
        collection_name: 'awx.awx'
```